strawberryfields/apps/graph/dense.py

```diff
@@ -0,0 +1,179 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+r"""
+Dense subgraph identification
+=============================
+
+**Module name:** :mod:`strawberryfields.future.apps.graph.dense`
```

Comment:
Can you remind me the reasoning behind the `graph` subpackage again? Is this because `apps` could potentially include multiple applications, which are not graph-based (and will go in another submodule?)

Reply:
That makes sense :+1: